protocol/x/sending/types/message_send_from_module_to_account.go

```diff
@@ -0,0 +1,36 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	codes "google.golang.org/grpc/codes"
+	status "google.golang.org/grpc/status"
+)
+
+var _ sdk.Msg = &MsgSendFromModuleToAccount{}
+
+// NewMsgWithdrawFromSubaccount constructs a `MsgWithdrawFromSubaccount` from an
+// `x/subaccounts` subaccount sender, an `x/bank` account recipient, an asset ID,
+// and a number of quantums.
+func NewMsgSendFromModuleToAccount(
+	authority string,
+	senderModuleName string,
+	recipient string,
+	coin sdk.Coin,
+) *MsgSendFromModuleToAccount {
+	return &MsgSendFromModuleToAccount{
+		Authority:        authority,
+		SenderModuleName: senderModuleName,
+		Recipient:        recipient,
+		Coin:             coin,
+	}
+}
+
+func (msg *MsgSendFromModuleToAccount) GetSigners() []sdk.AccAddress {
```

Comment:
Either implement this in a follow-up PR, or add unit test for `GetSigners` in this PR